frontend/src/pages/Common/LogIn/LogIn.tsx

```diff
@@ -68,38 +96,45 @@ export default function SignIn(props: LogInProps) {
           <Typography component="h1" variant="h5">
             Sign in
           </Typography>
-          <form className={classes.form} noValidate>
+          <form className={classes.form} onSubmit={handleSubmit} noValidate>
             <StyledTextField
               margin="normal"
               label="Email Address"
               name="email"
               autoComplete="email"
-              value={email}
-              onChange={(e) => setEmail(e.target.value)}
+              value={form.email}
+              onChange={handleFormChange}
               autoFocus
               data-testid="li-email"
+              error={invalidEmail}
+              helperText={invalidEmail && 'Email address is required!'}
             />
             <StyledTextField
               margin="normal"
               name="password"
               label="Password"
               type="password"
               autoComplete="current-password"
-              value={password}
-              onChange={(e) => setPassword(e.target.value)}
+              value={form.password}
+              onChange={handleFormChange}
               data-testid="li-password"
+              error={invalidPasswd}
+              helperText={invalidPasswd && 'Pasword is required!'}
             />
-            <Button
-              type="button"
-              fullWidth
-              variant="contained"
-              color="primary"
-              className={classes.submit}
-              onClick={handleSignInClick}
-              data-testid="li-button"
-            >
-              Sign In
-            </Button>
+            {!loading ? (
+              <Button
+                type="submit"
+                fullWidth
+                variant="contained"
+                color="primary"
+                className={classes.submit}
+                data-testid="li-button"
+              >
+                Sign In
+              </Button>
+            ) : (
+              <CircularProgress color="primary" className={classes.submit} />
+            )}
```

Comment:
In my opinion, you can move the loader inside the button and replace text with loader, while it's loading. If so, add prop 'disabled={loading}' to a button.

Reply:
this is the same convention as that in the register page. It actually depends on the business, project manager or The Boss #Konrad

Reply:
I should spend more time to find proper loading gif that could work inside disabled button so not sure if it's necessary for now